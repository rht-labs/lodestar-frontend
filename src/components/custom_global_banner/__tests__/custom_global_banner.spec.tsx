import React from 'react';
import { render } from '@testing-library/react';
import CustomGlobalBanner from '../custom_global_banner';

describe('custom global banner', () => {
  test('matches snapshot', () => {
    expect(
      render(<CustomGlobalBanner message="Hello, world!" color="white" />)
    ).toMatchSnapshot();
  });
});
