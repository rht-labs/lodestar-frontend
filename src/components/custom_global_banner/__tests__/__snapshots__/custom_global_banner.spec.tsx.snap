// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`custom global banner matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        style="background-color: white; height: 50px; width: 100%; display: flex; flex-direction: row; align-items: center; justify-content: center; padding: 0px 1.5rem;"
      >
        Hello, world!
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      style="background-color: white; height: 50px; width: 100%; display: flex; flex-direction: row; align-items: center; justify-content: center; padding: 0px 1.5rem;"
    >
      Hello, world!
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
